external/fv3kube/tox.ini

```diff
@@ -4,6 +4,8 @@ skipsdist = true
 
 [testenv]
 deps =
+    pytest==4.6.11
```

Comment:
These are now pinned to versions that are known to work in our other external packages that use tox. If left unpinned, tox now installs versions that conflict with python 3.8 that's in `fv3net`.